### DISCUSSION TITLE
Video and IFrame are not displayed

### DISCUSSION THREAD

**Discussion by Fla1337** [ReactUnity/core#55] [Q&A]
First of all thanks, designing the GUI with React is definitely easier than doing it with Unity. I just need help with the two things.

Video
I have now tried everything, but I just get error messages or the element is not displayed.

IFrame
I also need an <Iframe>. Have also done a few tests, but it is not displayed.

I also tried it with this.
<html content={htmlContent} />

Here as an example, one of my attempts.

Video
< video width="750" height="500" controls >
< source src={videosrc} type="video/mp4" />
< /video>

Iframe
< iframe width="420" height="345" src="https://website"></iframe>

Is there a solution or is it currently not possible.

I am currently using Unity 2021.3.1f1 LTS

**Comment by gkurt** [comment]
Hello. ReactUnity is not an in-game browser. It does not have capability to render websites like browsers do. So `iframe` is not and will not be supported. You can check a 3rd party library for that. For example https://assetstore.unity.com/packages/tools/gui/embedded-browser-55459#description

There is a video component but it does not work like browsers. It uses Unity's VideoPlayer component. Usage is like:

```
<video source={videosrc} />
```
